src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -656,6 +690,24 @@ public void onRmdChunkOutput(RmdChunkOutputEvent event)
 
       String chunkId = event.getOutput().getChunkId();
       
+      // ignore requests performed to initialize satellite chunks
+      if (satelliteChunkRequestIds_.contains(event.getOutput().getRequestId()))
+      {
+         int mode = queue_.getChunkExecMode(chunkId);
+         
+         forwardEventToSatelliteChunk(
+            docUpdateSentinel_.getId(),
+            chunkId,
+            new ChunkSatelliteShowChunkOutputEvent(
+                  event.getOutput(),
+                  mode,
+                  NotebookQueueUnit.EXEC_SCOPE_PARTIAL,
```

Comment:
You can pick up the correct scope from the `queue_` as well (but don't bother unless the distinction is relevant to your use case).